docs/_documentations/debugging.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Debugging in Codewind
+description: Debugging in Codewind
+keywords: debug, mode, applications, application, project, menu, view, attaching, attach, actions, action, run, restart in debug mode, attach debugger, debug process, debugging process
+duration: 1 minute
+permalink: debugging
+type: document
+---
+
+# Debugging in Codewind
+
+Complete the following steps to enable debugging in Codewind applications:
+
+1. In the Codewind view, right-click a project and select **Restart in Debug Mode** from the menu.
+2. Wait for the project state to switch to **Debugging**. The first time that the project switches to **Debugging**, the debugger is attached.
```

Comment:
Edit (slightly different from my original outline):
Wait for the project state to switch to **Debugging**. Once it switches to **Debugging** mode, the debugger is attached.